edalize/symbiyosys.py

```diff
@@ -205,7 +246,6 @@ def _get_read_flags(self):
                 "-D{}={}".format(key, self._param_value_str(value))
                 for key, value in self.vlogdefine.items()
             ]
-            + ["-I{}".format(inc) for inc in self.incdirs]
```

Comment:
No longer do includes with flags. These are not honoured by symbiyosys (`read -help` also doesn't list this form)